src/main/java/com/github/standobyte/jojo/action/non_stand/PillarmanWindCloak.java

```diff
@@ -20,11 +19,15 @@ public PillarmanWindCloak(PillarmanAction.Builder builder) {
         mode = Mode.WIND;
     }
     
-    @Override
-    public void holdTick(World world, LivingEntity user, INonStandPower power, int ticksHeld, ActionTarget target, boolean requirementsFulfilled) {
-        if (!world.isClientSide() && requirementsFulfilled) {
-        	user.addEffect(new EffectInstance(Effects.INVISIBILITY, 5, 0, false, false));
-            user.addEffect(new EffectInstance(ModStatusEffects.SUN_RESISTANCE.get(), 5, 0, false, false));
+    public static void windEffect(LivingEntity user, IParticleData particles, int intensity) {
+        if (user.isInvisible() || user.hasEffect(Effects.INVISIBILITY) || user.hasEffect(ModStatusEffects.FULL_INVISIBILITY.get()))
```

Comment:
These particles are still supposed to appear when the player is exiting the wind cloak, but is still technically under the invisibility effect, this condition breaks that

Reply:
almost dont realize it. as if "Wind Cloak" shouldnt output such saturated particles to be honest. at least when there is already have an effect